.travis.yml

```diff
@@ -15,3 +15,12 @@ script:
 
 after_success:
     - slather
+
+notifications:
+    email: false
```

Comment:
I never received it so I assumed it was the default off. But I guess it's the other way.